src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/Touch/PlatformTouchEffect.android.cs

```diff
@@ -45,9 +46,13 @@ public class PlatformTouchEffect : PlatformEffect
 			&& accessibilityManager.IsEnabled
 			&& accessibilityManager.IsTouchExplorationEnabled;
 
-		bool IsForegroundRippleWithTapGestureRecognizer => View.Foreground == ripple
-			&& Element is XView view
-			&& view.GestureRecognizers.Any(gesture => gesture is TapGestureRecognizer);
+		bool IsForegroundRippleWithTapGestureRecognizer
+			=> ripple != null &&
+				ripple.IsAlive() &&
+				View.IsAlive() &&
+				View.Foreground == ripple &&
+				Element is XView view &&
+				view.GestureRecognizers.Any(gesture => gesture is TapGestureRecognizer);
```

Comment:
The `IsAlive` method already check for `null`
```suggestion
			=> ripple.IsAlive() &&
				View.IsAlive() &&
				View.Foreground == ripple &&
				Element is XView view &&
				view.GestureRecognizers.Any(gesture => gesture is TapGestureRecognizer);
```

Reply:
@pictos but null-check linter rule doesn't know about it :) So have to do it here as well

Reply:
:(